dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -625,6 +629,144 @@ public Response deleteAllSuccess(@Context final HttpServletRequest request,
                 "Removing bundles in a separated process, the result of the operation will be notified")).build();
     } // deleteAllSuccess.
 
+
+    @Path("/_download/{bundleId}")
+    @GET
+    @JSONP
+    @NoCache
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    public final Response downloadBundle(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @PathParam("bundleId") final String bundleId) {
+
+        final InitDataObject initData =
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6239861)